docs/2.9.0/docs/tools/daml-shell/index.rst

```diff
@@ -0,0 +1,454 @@
+.. Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
```

Comment:
```suggestion
.. SPDX-License-Identifier: Apache-2.0

.. enterprise-only::

```

Reply:
Missed this in the other reviews -- doesn't Daml Shell require an enterprise license? 

This directive adds the enterprise license disclaimer, like the one on [this page](https://docs.daml.com/canton/usermanual/domains/ethereum.html) (and many others)

It looks like this in local preview:

<img width="715" alt="Screenshot 2024-02-15 at 11 44 56 AM" src="https://github.com/digital-asset/docs.daml.com/assets/139921049/cb71ee4d-6a64-4de4-bbba-bded4f3ee5d9">


👉 (but if I'm wrong about Daml Shell being an enterprise feature, reject this change!) 🙅

Reply:
It does require an enterprise license.